### DISCUSSION TITLE
how to make mpv ignore images when opening a folder

### DISCUSSION THREAD

**Discussion by rezad1393** [mpv-player/mpv#14636] [Q&A]
I use mpv for watching videos.
when I right click on a folder in kde file manager and click open with mpv, video files in that folder and its subfolders are queued for playback.
but if there is a pictures folders in that folder, pictuers are also added to the queue.

how can I make mpv ignore pictures and only load videos?

**Comment by llyyr** [comment]
https://github.com/occivink/mpv-scripts/blob/master/scripts/blacklist-extensions.lua

**Reply by rezad1393** [reply]
it is a bit complicated for me.
how do I for example block jpeg files?

**Reply by llyyr** [reply]
Check the readme of the repo, it has instructions for how to use those scripts

**Reply by rezad1393** [reply]
it just says:
The script doesn't do anything by default, you need to copy script-opts/blacklist_extensions.conf and modify it to your liking.

when I add     blacklist=jpg, and open folder again it still loads them.

**Reply by Fuwiz** [reply]
> it just says: The script doesn't do anything by default, you need to copy script-opts/blacklist_extensions.conf and modify it to your liking.
> 
> when I add blacklist=jpg, and open folder again it still loads them.

Try this: https://github.com/mpv-player/mpv/issues/2595#issuecomment-1762776452

**Comment by lazypingu** [comment]
Maybe `--directory-filter-types=video` can be used.
The options `--directory-mode` and `--autocreate-playlist` might be also of interest.
